app/presenters/collection_presenter.rb

```diff
@@ -0,0 +1,28 @@
+class CollectionPresenter
```

Comment:
@RamilGilmanov, +1 on naming them `BasePresenter` and `BaseCollectionPresenter`. In that case we distinguish collection presenter and emphasize that these are base presenters used as parent for all application presenters.